sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/socket.scala

```diff
@@ -130,16 +130,7 @@ class TextSocketSource(host: String, port: Int, includeTimestamp: Boolean, sqlCo
 
     val rdd = sqlContext.sparkContext.parallelize(rawList).map(
         v => InternalRow(UTF8String.fromString(v._1), v._2.getTime()))
```

Comment:
May I ask to replace `v` with `case...` as follows? IMHO That would make things easier to read.

```
    val rdd = sqlContext.sparkContext.
      parallelize(rawList).
      map { case (v, ts) => InternalRow(UTF8String.fromString(v), ts.getTime) }
```

Reply:
@joseph-torres this is a nit but a good suggestion.

Reply:
Done